### PR TITLE
[awsping] add package

### PR DESCRIPTION
**Pull request by osterman**
## what
* Add `awsping` package

## why
* Easily determine the nearest AWS data center

### CODE REVIEW

File: vendor/awsping/Makefile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+export VENDOR ?= ekalinin
+export DOWNLOAD_URL ?= $(PACKAGE_REPO_URL)/releases/download/$(PACKAGE_VERSION)/$(PACKAGE_NAME).$(OS).$(ARCH).tgz
+export APK_BUILD_TEMPLATE ?= APKBUILD.github-binary
+
+install:
+	$(CURL) -o - $(DOWNLOAD_URL) | tar --wildcards -zxO awsping > $(INSTALL_PATH)/awsping
+	chmod +x $(INSTALL_PATH)/awsping
+
+test:
+	$(PACKAGE_EXE) --version
```
Comment (aknysh) [inline vendor/awsping/Makefile]: @osterman 
not sure if `--version` is the correct command.

https://github.com/ekalinin/awsping/blob/master/main.go#L26

uses `v` and the module does not use any CLI libraries, but just parses the arguments directly.
```suggestion
	$(PACKAGE_EXE) -v
```

File: vendor/awsping/Makefile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+package/prepare::
+	mv src/awsping_v$(PACKAGE_VERSION)_$(OS)_$(ARCH)/$(PACKAGE_EXE) src
```
Comment (aknysh) [inline vendor/awsping/Makefile]: is this correct?

**Comment by Nuru** [comment]
Not enough interest in this

**Pull request closed without merging**